packages/squiggle-lang/src/library/registry/frTypes.ts

```diff
@@ -123,6 +123,56 @@ export const frTuple2 = <T1, T2>(
   };
 };
 
+export function frTuple<T1, T2>(
+  type1: FRType<T1>,
+  type2: FRType<T2>
+): FRType<[T1, T2]>;
+
+export function frTuple<T1, T2, T3>(
+  type1: FRType<T1>,
+  type2: FRType<T2>,
+  type3: FRType<T3>
+): FRType<[T1, T2, T3]>;
+
+export function frTuple<T1, T2, T3, T4>(
+  type1: FRType<T1>,
+  type2: FRType<T2>,
+  type3: FRType<T3>,
+  type4: FRType<T4>
+): FRType<[T1, T2, T3, T4]>;
+
+export function frTuple<T1, T2, T3, T4, T5>(
+  type1: FRType<T1>,
+  type2: FRType<T2>,
+  type3: FRType<T3>,
+  type4: FRType<T4>,
+  type5: FRType<T5>
+): FRType<[T1, T2, T3, T4, T5]>;
+
+export function frTuple(...types: FRType<unknown>[]): FRType<any> {
```

Comment:
I had trouble making "FRType<any> " into FRType<unknown[]>

Reply:
That's fine, I'm not sure why it worked for me in my Discord example, tbh, seems hard to fix.